src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -794,7 +887,7 @@ bool HMICapabilities::load_capabilities_from_file() {
         }
 
         if (display_capabilities_so.keyExists(hmi_response::text_fields)) {
-          uint32_t len =
+          const uint32_t len =
```

Comment:
@LevchenkoS should it be named kLen? 
Please, check coding style with const names